index.js

```diff
@@ -18,20 +16,26 @@ app.listen(PORT, function(req, res){
 	console.log(`Server is running @ ${PORT}`);
 })
 
-
-
 app.get('/getWithYear', malaria);
 
-
 function malaria(req, res){
-	request(url, function(err, response, body){
+		var options ={
+			query: req.query.query,
+			start: req.query.start,
+			end: req.query.end,
+			sort: 'CITED desc',
+			pageSize:1,
+			synonym: true,
+			format: 'json'
+		}
+		request({url:reqUrl, qs:options }, function(err, response, body){
```

Comment:
@Firnaa5 Is it working? I doubt it! 
` ( FIRST_PDATE:[Some From date TO Some To Date])` this has to come in the URL that you are sending to request. 
I think you understood it wrong. the API `/getWithYear` can receive query strings like `start, end` etc.. Since this is your API and you can have inputs as you want. But for  the API `https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=malaria( FIRST_PDATE:[2013-01-01 TO 2013-12-31])&format=json&synonym=true&pageSize=20&sort=CITED desc` should not change in the format.

Reply:
@Firnaa5 It's good that you've used `options` JSON. but you have missed seeing that for the key `query` in the queryString the value is `malaria( FIRST_PDATE:[2013-01-01 TO 2013-12-31])` So first you build up this string first then pass it on to the `query` property in the options JSON. which will negate the use of properties like `start , end` from the same JSON

Reply:
@Firnaa5 What about the change I requested here?